mindocr/losses/rec_loss.py

```diff
@@ -69,3 +69,17 @@ def construct(self, logits: Tensor, labels: Tensor) -> Tensor:
         logits = ops.reshape(logits, (-1, num_classes))
         labels = ops.reshape(labels, (-1,))
         return self.criterion(logits, labels)
+
+
+class MasterLoss(LossBase):
```

Comment:
looks identical to `AttentionLoss` (except `ignore_index`). Can just combine them?

Reply:
right, it is combined now.